### DISCUSSION TITLE
Running Playwright (or the stack) in Docker

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#764] [Ideas]
Running visual regression tests on macOS saves the snapshots as `*-chromium-darwin.png`. During deployment they are compared to `*-chromium-linux.png`, so all visual regression tests fail.

Would be nice to have Playwright (or the stack) run in the same environment in both cases, to solve problems like this.

**Comment by fredericrous** [comment]
could this be fixed by just documentation, in the spirit of https://github.com/epicweb-dev/epic-stack/issues/662 ? You will need ofc to add the `-v` flag to attach a volume.
I personally prefer to run the apps I develop on my machine and not in docker containers. this includes tests, dev server.. (that's why I put a thumbs down)
